src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -698,7 +698,11 @@ private void OnUpdateGraphCompleted(AsyncTask task)
                 var node = workspace.Nodes.FirstOrDefault(n => n.GUID == guid);
                 if (node == null)
                     continue;
+
+                // Block Infos updates during the many errors/warnings/notifications added here
+                // InfoBubbles will be updated on NodeViewModel's EvaluationCompleted handler.
                 using (node.PropertyChangeManager.SetPropsToSuppress(nameof(NodeModel.Infos), nameof(NodeModel.State)))
+                using (Disposable.Create(() => { node.BlockInfoBubbleUpdates = true; }, () => { node.BlockInfoBubbleUpdates = false; }))
```

Comment:
I haven't refreshed my memory on the `PropertyChangeManager` and how it suppresses property-changed events, but my question is are both this and the `blockInfoBubbleUpdates` required? How are these different if both ensure the UI updates on the info bubbles happen at the end of the graph execution? In other words, aren't the updates batched for the end of the graph execution already (with the introduction of `PropertyChangeManager`?